cfpb/consumerfinance.gov
# Resize filterable list control expandable when inner multiselects change

Adding/removing items from multiselects within filterable list controls causes the height of the expandable's content to increase/decrease. When it increases, the bottom of the content gets cut off.

This PR recalculates the height of the expandable when the multiselect's options are changed.

See https://github.local/CFGOV/platform/issues/3474 for more info.

## Additions

- `selectionsUpdated` event listener.

## Changes

- Moved the refresh code into a separate function to avoid repetition.

## How to test this PR

1. Go to https://www.consumerfinance.gov/about-us/blog/ and add three or four items in the `Topic` multiselect. Notice how the apply button gets cut off until you click away and close the multiselect.
2. Do the same at http://0.0.0.0:8000/about-us/blog/ and the apply button should remain in view while the multiselect is still open.

## Screenshots


## Notes and todos

-

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
